x-pack/plugins/security/server/config.ts

```diff
@@ -200,6 +200,23 @@ export const ConfigSchema = schema.object({
   }),
   audit: schema.object({
     enabled: schema.boolean({ defaultValue: false }),
+    appender: schema.maybe(coreConfig.logging.appenders),
+    ignore_filters: schema.conditional(
+      schema.siblingRef('appender'),
+      coreConfig.logging.appenders,
```

Comment:
does it mean that `ignore_filters` cannot be applied to default appenders?

Reply:
This will be removed when the new audit logger goes out of beta. While it is in beta there is no default appender.

Setting `pack.security.audit.enabled=true` enables the legacy audit logger as per current behaviour.
Setting `pack.security.audit.enabled=true` in addition to setting an `pack.security.audit.appender` enables the new beta audit logger.

That's why `ignore_filters` are conditional to `appenders` since they don't work with the legacy audit logger.